### DISCUSSION TITLE
Youtube på events virker ikke

### DISCUSSION THREAD

**Discussion by MartinQwist** [xooom-community/public#9] [Bugs] [answered]
Når man indsætter et youtube link på et event, så får man en fejlmelding, og linket/videoen vises ikke på eventet. 

![image](https://github.com/user-attachments/assets/b7a4c708-3a27-4eef-8342-bbc2d5cd47a8)

**Comment by antonlund18** [comment, marked as answer]
Det er en fejl fra YouTube, da de ikke tillader at denne URL bruges på andre domæner end deres egne.

I stedet har de en speciel URL, man kan bruge for at indlejre videoen på andre sider (den burde indeholde `/embed/` i URL'en). Fra YouTube -> Del -> Indlejr Video:
![image](https://github.com/user-attachments/assets/8caadcb1-c964-44b4-9ce7-bad8e2efac8a)

Med det sagt, så tror jeg ikke systemet understøtter at vise videoer på portalen endnu?

**Reply by MartinQwist** [reply]
OK. Kan vi fjerne muligheden indtil den virker?

**Reply by antonlund18** [reply]
Yes, jeg har skjult den.
